README.md

```diff
@@ -10,18 +10,40 @@ The humble beginnings of a Nim library similar to web3.[js|py]
 ## Installation
 
 You can install the developement version of the library through nimble with the following command
+
 ```
 nimble install https://github.com/status-im/nim-web3@#master
 ```
 
+## Development
+
+You should first run `./simulator.sh` which runs `ganache-cli`
+
+This creates a local simulated Ethereum network on your local machine and the tests will use this for their E2E processing
+
+### Interaction with nimbus-eth2
+
+This repo relies heavily on parts of the `nimbus-eth2` repo.
```

Comment:
in which ways? ie it relies on there being a working nim setup, and nimbus-eth2 is one of the easier ways to get to one but..